From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: rajeshvenkata <rajesh.venkat.p@gmail.com>
Date: Wed, 28 Jan 2026 00:00:00 +0000
Subject: [PATCH 1/1] Disable secondstar service

EKS-A does not use the secondstar (SSH over serial) service.
Disable it by default in the helm chart values.

Signed-off-by: rajeshvenkata <rajesh.venkat.p@gmail.com>
---
 helm/tinkerbell/values.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/helm/tinkerbell/values.yaml b/helm/tinkerbell/values.yaml
index 1234567..abcdefg 100644
--- a/helm/tinkerbell/values.yaml
+++ b/helm/tinkerbell/values.yaml
@@ -28,7 +28,7 @@ deployment:
       backendKubeQPS: 100
       bindAddr: ""
       enableCRDMigrations: true
       enableRufioController: true
-      enableSecondstar: true
+      enableSecondstar: false
       enableSmee: true
       enableTootles: true
       enableTinkServer: true
-- 
2.40.1